app/views/step_by_step_pages/change_notes.html.erb

```diff
@@ -0,0 +1,48 @@
+<%
+  links = [
+    {
+      text: 'Step by step publisher',
+      href: step_by_step_pages_path
+    },
+    {
+      text: @step_by_step_page.title,
+      href: @step_by_step_page
+    },
+    {
+      text: 'Record a new change note or review prior changes'
+    }
+  ]
+%>
+
+<%= render 'shared/steps/step_breadcrumb', links: links %>
+
+<%= render 'shared/steps/page_title', links: links %>
+
+<%= render 'shared/steps/heading',
+  step_nav: @step_by_step_page.title,
+  action: 'Record a new change note or review prior changes'
+%>
+
+<%= render 'nav', step_by_step_page: @step_by_step_page, active: "change-notes" %>
+
+<%
+  left_col = "col-md-7"
+  right_col = "col-md-5"
+%>
+
+<%= form_for(InternalChangeNote.new, url: internal_change_notes_create_path) do |form| %>
```

Comment:
Ideally the view should have the same name as the controller. So this file should be called `internal_change_note.html.erb`.

You should be able to create a new instance of the class (`InternalChangeNote.new`) in the controller method, and just refer to it here.